Update `CHANGELOG` for 3.32.5, 3.32.6 stable hotfix releases (#171891)
flutter/flutter: CHANGELOG.md

-->
## Flutter 3.32 Changes

> ### [3.32.6](https://github.com/flutter/flutter/releases/tag/3.32.6)

Reviewer: https://github.com/flutter/flutter/blob/main/docs/releases/Hotfix-Documentation-Best-Practices.md
“When $scenario [on $platform], $problem_description”

> - [flutter/171106](https://github.com/flutter/flutter/pull/171106) - Fixes a "Null check operator used on a null value" crash when a scroll view contains a `LayoutBuilder`.

Reviewer: ```suggestion
- [flutter/171106](https://github.com/flutter/flutter/pull/171106) - When a scrollview contains a `LayoutBuilder` on any platform prevent null check crash .
```

> - [flutter/171239](https://github.com/flutter/flutter/pull/171239) - Fixes an issue in the Impeller Vulkan back end that can cause crashes when transitioning between different Android activities that use Flutter.

Reviewer: ```suggestion
- [flutter/171239](https://github.com/flutter/flutter/pull/171239) - When using Impeller + Vulkan and transitioning between activities that use flutter on Android prevent a crash.
```

> - [flutter/171737](https://github.com/flutter/flutter/pull/171737) - Fixes full app crash when backgrounding and then bringing an app back up on Android 10-13 using an app with platform views.

Reviewer: ```suggestion
- [flutter/171737](https://github.com/flutter/flutter/pull/171737) - When using platform views on android 10-13 (api 29-33) prevent app crash when backgrounding and then foregrounding app.
```
